stix_shifter_modules/cybereason/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,2299 @@
+    "macAddressFormat": [
+      {
+        "key": "mac-addr.value",
+        "object": "mac"
+      },
+      {
+        "key": "ipv4-addr.resolves_to_refs",
+        "object": "src_ip",
+        "references": [
+          "mac"
+        ]
+      }
+    ],
+    "id": {
+      "key": "x-oca-event.interface_id",
+      "object": "event"
+    },
+    "proxies": {
+      "key": "ipv4-addr.extensions.x-cybereason-networkinterface.proxies",
+      "object": "src_ip"
+    }
+  },
+  "Machine": {
+    "elementDisplayName": [
+      {
+        "key": "x-oca-asset.name",
+        "object": "asset"
+      }
+    ],
+    "mountPoints": {
+      "key": "x-oca-asset.mount_points",
+      "object": "asset"
+    },
+    "processes": {
+      "key": "x-oca-asset.processes",
+      "object": "asset"
+    },
+    "services": {
+      "key": "x-oca-asset.services",
+      "object": "asset"
+    },
+    "logonSessions": {
+      "key": "x-oca-asset.logon_sessions",
+      "object": "asset"
+    },
+    "hasRemovableDevice": {
+      "key": "x-oca-asset.has_removable_device",
+      "object": "asset"
+    },
+    "timezoneUTCOffsetMinutes": {
+      "key": "x-oca-asset.timezone",
+      "object": "asset"
+    },
+    "osVersionType": {
+      "key": "x-oca-asset.version_type",
+      "object": "asset"
+    },
+    "platformArchitecture": {
+      "key": "x-oca-asset.platform",
+      "object": "asset"
+    },
+    "mbrHashString": {
+      "key": "x-oca-asset.mbr_hash",
+      "object": "asset"
+    },
+    "osType": {
+      "key": "x-oca-asset.os_type",
+      "object": "asset"
+    },
+    "domainFqdn": [
+      {
+        "key": "domain-name.value",
+        "object": "machine_domain"
+      },
+      {
+        "key": "x-oca-asset.domain_name",
+        "object": "asset"
+      }
+    ],
+    "ownerOrganization": {
+      "key": "x-oca-asset.owner_organization",
+      "object": "asset"
+    },
+    "pylumId": {
+      "key": "x-oca-asset.pylum_id",
+      "object": "asset"
+    },
+    "adSid": [
+      {
+        "key": "user-account.user_id",
+        "object": "user-machine"
+      },
+      {
+        "key": "x-oca-asset.ad_sid",
+        "object": "asset"
+      }
+    ],
+    "adOU": [
+      {
+        "key": "autonomous-system.name",
+        "object": "machine_as"
+      },
+      {
+        "key": "x-oca-asset.ad_ou",
+        "object": "asset"
+      }
+    ],
+    "adOrganization": [
+      {
+        "key": "autonomous-system.name",
+        "object": "machine_as"
+      },
+      {
+        "key": "x-oca-asset.ad_organization",
+        "object": "asset"
+      }
+    ],
+    "adCanonicalName": {
+      "key": "x-oca-asset.canonical_name",
+      "object": "asset"
+    },
+    "adCompany": [
+      {
+        "key": "x-oca-asset.ad_company",
+        "object": "asset"
+      },
+      {
+        "key": "autonomous-system.name",
+        "object": "machine_as"
+      }
+    ],
+    "adDNSHostName": {
+      "key": "x-oca-asset.domain_name",
```

Comment:
This should just be a domain-name object

Reply:
Changes has been done as per the suggestion  in to_stix_map.json file